src/cpp/benders/benders_by_batch/BendersByBatch.cpp

```diff
@@ -46,6 +47,10 @@ void BendersByBatch::InitializeProblems() {
       problem_count++;
     }
   }
+
+  // if (Rank() == rank_0) {
+  SetSubproblemsVariablesIndex();
+  // }
```

Comment:
...

Reply:
Maybe add a comment to explain what happens here